src/ring_theory/polynomial/hermite.lean

```diff
@@ -6,6 +6,8 @@ Authors: Luke Mantle
 
 import data.polynomial.derivative
 import data.nat.parity
+import analysis.special_functions.exp
+import analysis.special_functions.exp_deriv
```

Comment:
This is a comparatively heavy import; it might be a good indication that these results deserve a new file. No need to do that immediately though

Reply:
I think it would be better to move this to a new file now. Costs little, and helps save on import creep.

Reply:
OK. What should the filename be? `hermite_gaussian.lean`?

Reply:
Yes. Or maybe even start a `hermite` folder:

* `polynomial/hermite/basic.lean`
* `polynomial/hermite/gaussian.lean`